node_modules/oae-core/editgroup/js/editgroup.js

```diff
@@ -101,6 +110,7 @@ define(['jquery', 'oae.core'], function($, oae) {
          */
         var setUpEditGroupModal = function() {
             $(document).on('click', '.group-trigger-editgroup', function() {
+                focusSelector = $(this).attr('data-editgroup-focus');
```

Comment:
I'm accounting for the potential that the editgroup modal can be triggered by some means other than a `.group-trigger-editgroup` class. Is that worthwhile? If not, then I think I can follow through with this feedback and cleanup the `focusSelector = null;` handler above as well.